app/controllers/api/integrations_controller.rb

```diff
@@ -36,7 +37,7 @@ def update
 
       if @proxy.send_api_test_request!
         onboarding.bubble_update('api')
-        done_step(:api_sandbox_traffic) if ApiClassificationService.test(@proxy.api_backend).real_api?
+        done_step(:api_sandbox_traffic) if @proxy.api_backend && ApiClassificationService.test(@proxy.api_backend).real_api?
```

Comment:
`@proxy.api_backend.present?`

Reply:
we shouldn't be calling `@proxy.api_backend` twice anyway 😄 better to assign it to a variable